extensions/amp-social-share/1.0/social-share-config.js

```diff
@@ -48,26 +48,34 @@ const BUILTINS = {
       'text': 'TITLE',
       'url': 'CANONICAL_URL',
     }),
+    'canChangeColor': true,
```

Comment:
I think these will never be out of sync. So, we can just `canCustomize: boolean`.

Reply:
Thanks, updated!